updater/dart_bindings/lib/src/bindings.dart

```diff
@@ -49,6 +52,12 @@ class AppParameters extends ffi.Struct {
     calloc.free(config.ref.base_version);
     calloc.free(config.ref.channel);
     calloc.free(config.ref.update_url);
+    // Free all paths in original_libapp_path.
+    var i = 0;
+    while (config.ref.original_libapp_path[i].address != 0) {
```

Comment:
Is it guaranteed that the last entry will be 0?

Reply:
Was reading about it and it looks like lists are null-terminated and the way to determine this is by checking `pointer.address == 0`.